code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -366,16 +366,15 @@
 //This proc is called when you want to place an item into the storage item.
 /obj/item/weapon/storage/attackby(obj/item/W, mob/user, params)
 	..()
-
+	. = 1 //no afterattack

```

Comment:
we always return 1 even if the insertion into the storage item fails, because we never want afterattack to be called.